graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -26,6 +42,58 @@ def _is_fractional(selectivity):
     return selectivity.kind == FRACTIONAL_SELECTIVITY
 
 
+def _get_intersection_of_IntegerIntervals(interval_a, interval_b):
```

Comment:
Let's stick to `snake_case` for naming. You could even make this function be named `_get_intersection_of_intervals` and even that would be okay.